index.html

```diff
@@ -175,36 +238,42 @@ <h1>About me</h1>
 
             <hr id="about-hr"/>
 
-            <ul id="skills-block">
-                <div class="skill-title-block">
-                    <div id="languages-logo" class="about-logo"></div>
-                    <h2 class="skill-title">Languages</h2>
-                </div>
-                <li>JavaScript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
+            <div id="skills-block">
+                <ul>
+                    <div class="skill-title-block">
+                        <div id="languages-logo" class="about-logo"></div>
+                        <h2 class="skill-title">Languages</h2>
+                    </div>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
```

Comment:
- [ ] I noticed a few differences between your 3 columns Language, Frameworks, and Skills, make sure that to fix the design and make your design match with the chosen Figma template :smile: 

#### Your design
![Screenshot from 2022-09-29 21-18-38](https://user-images.githubusercontent.com/6615394/193144558-2629a6ad-f7e2-4cd3-ae5d-2c1ee83d93b5.png)

#### Figma design
![Screenshot from 2022-09-29 21-18-08](https://user-images.githubusercontent.com/6615394/193144562-cf266fd3-9078-45a2-9cef-9ca39c250750.png)